VideoWeb/VideoWeb/ClientApp/src/app/on-the-day/judge-hearing-list/judge-hearing-table.component.ts

```diff
@@ -20,12 +21,12 @@ export class JudgeHearingTableComponent implements OnInit {
 
     @Output() selectedConference = new EventEmitter<ConferenceForHostResponse>();
 
-    constructor(private logger: Logger) {}
+    constructor(private logger: Logger, private hearingVenueFlagsService: HearingVenueFlagsService) {}
 
     ngOnInit() {
         this.hearings = this.conferenceForHostResponse.map(c => new JudgeHearingSummary(c));
         const last = this.hearings.pop();
-
+        this.hearingVenueFlagsService.setHearingVenueIsScottish(false);
```

Comment:
Any reason this can't go in the judge-hearing-list.component?

Reply:
no, i cant remember why I put it in the hearing table component, for the partcipants it appears to be in the table component, so I can try to do that and see if it works as expected